detekt/src/main/kotlin/potentialbugs/StorageNeedsExpose.kt

```diff
@@ -39,8 +39,10 @@ class StorageNeedsExpose(config: Config): SkyHanniRule(config) {
                 // Valid reasons to not have the @Expose annotation on a config option:
                 //  - Has the ConfigEditorInfoText annotation
                 //  - Has the ConfigEditorButton annotation
+                //  - Has the Transient annotation
                 if(property.hasAnnotation("ConfigEditorInfoText")) return
                 if(property.hasAnnotation("ConfigEditorButton")) return
+                if(property.hasAnnotation("Transient")) return
```

Comment:
if detekt saw this formatting it would be very upset :(

Reply:
![grafik](https://github.com/user-attachments/assets/336d01de-0a8d-4752-9328-2a7a65a94c24)